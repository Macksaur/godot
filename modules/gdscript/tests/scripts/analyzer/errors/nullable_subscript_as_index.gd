class A:
    var index: int?
    func get_nullable_instance() -> A?:
        return A.new()
func test():
    var foo: Array[Array] = [[]]
    var a_instance := A.new()
    foo[0][a_instance.get_nullable_instance()?.index]
