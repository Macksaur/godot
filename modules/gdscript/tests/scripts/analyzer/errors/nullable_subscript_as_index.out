GDTEST_ANALYZER_ERROR
Potentially unsafe for the nullable expression "index" to be used as an index.
